tests/requests/test_user_requests.py

```diff
@@ -73,4 +75,38 @@ def test_it_cant_register_a_user_without_email(db, cleanup):
     json={"email": email, "password": password})
     resp = response.json()
     assert resp['detail'][0]['msg'] == 'value is not a valid email address'
-    
\ No newline at end of file
+
+def test_it_can_update_a_user(db, cleanup):
```

Comment:
We should add an assertion for if the user doesn't exist in the system, ie `assert update_response.status_code == 404`

Reply:
Good catch! Just update test for the sad path